.travis.yml

```diff
@@ -36,6 +36,17 @@ matrix:
         - pycodestyle scipy benchmarks/benchmarks
         - |
           grep -rlIP '[^\x00-\x7F]' scipy | grep '\.pyx\?' | sort > unicode.out; grep -rlI '# -\*- coding: \(utf-8\|latin-1\) -\*-' scipy | grep '\.pyx\?' | sort > coding.out; comm -23 unicode.out coding.out > test_code.out; cat test_code.out;  test \! -s test_code.out
+    - python: 3.7
+      if: type = pull_request
+      arch: arm64
+      dist: bionic
+      env:
+        - TESTMODE=fast
+        - COVERAGE=
+        # Need a test with most recent Python version where we build from an
+        # sdist (uses pip build isolation), to check that the version we
+        # specify in pyproject.toml (will be older than --pre --upgrade) works
```

Comment:
This comment can be removed, it's copied from the other `USE_SDIST=1` test, no need to have it twice.

I am +1 on keeping `USE_SDIST=``.